test/framework/e2e_framework.go

```diff
@@ -446,17 +452,25 @@ func (f *E2eFramework) DeployK8sClientConfigs(namespace string) error {
 		return err
 	}
 
+	destCfg := fmt.Sprintf(*configFileTemplate, 0)
+	kubeconfigs := filepath.Join(baseDir, *configDir)
+	inClusterKubeconfigs := filepath.Join(baseDir, *inClusterConfigDir)
```

Comment:
I need to look more closely at this, but I want to point out that the in-cluster kubeconfig is only needed for a local cluster like kind.